src/index.html

```diff
@@ -15,30 +15,11 @@ <h1>Loaderboard</h1>
             <div class="display-score score">
                 <div class="description">
                     <h2>Recents scores</h2>
-                    <button type="button">Refresh</button>
+                    <button type="button" id="refresh">Refresh</button>
                 </div>
-                <ul id="items">
-                    <li class="itemScore">
-                        <h3>Ronaldo :</h3>
-                        <p>700</p>
-                    </li>
-                    <li class="itemScore">
-                        <h3>Messi :</h3>
-                        <p>600</p>
-                    </li>
-
-                    <li class="itemScore">
-                        <h3>Mbappe :</h3>
-                        <p>400</p>
-                    </li>
-
-                    <li class="itemScore">
-                        <h3>Neymar :</h3>
-                        <p>200</p>
-                    </li>
-                </ul>
+                <ul id="items"></ul>
             </div>
-    
+
             <div class="add-score score">
                 <h2>Add your score</h2>
```

Comment:
- [ ] Your App is working fine 👍 
![Capture](https://user-images.githubusercontent.com/63972670/225272454-85400b36-2e18-49e4-aa40-d90c420d5a0c.PNG)